packages/server/src/utils/index.ts

```diff
@@ -299,6 +299,8 @@ export const buildFlow = async (
         exploredNode[startingNodeIds[i]] = { remainingLoop: maxLoop, lastSeenDepth: 0 }
     }
 
+    const initializedNodes: Set<string> = new Set()
+    const nonDirectedGraph = constructGraphs(reactFlowNodes, reactFlowEdges, { isReversed: true }).graph
```

Comment:
can we rename to `reversedGraph`?

Reply:
I copied it here.

https://github.com/FlowiseAI/Flowise/blob/13ee0d0c7b4c325100ec8dbd3919a26dd0db8893/packages/server/src/index.ts#L2025

Reply:
ah we should rename both as well, as its a reversed graph, not a non directed graph